src/main/java/net/imglib2/algorithm/componenttree/mser/MserTree.java

```diff
@@ -105,7 +105,8 @@ public final class MserTree< T extends Type< T > > implements ComponentForest< M
 	 * {@link #buildMserTree(RandomAccessibleInterval, RealType, long, long, double, double, ImgFactory, boolean)}
 	 * using an {@link ArrayImgFactory} or {@link CellImgFactory} depending on
 	 * input image size.
-	 * 
```

Comment:
Opinions on whether these should be Ops? I'm not totally sure what they're used for...

Reply:
https://github.com/imglib/imglib2-tests/blob/180e0f988c572518e0b35bedc558186a5b79be8a/src/test/java/net/imglib2/algorithm/componenttree/mser/MserTreeExample.java

Reply:
Concepts of level sets, mathematical morphology, and connected component analysis are very related. We have CCA currently in `labeling`, but other morphology operators in `morphology`. What about [level sets](https://imagej.net/plugins/level-sets)? FWIW, I checked scikit-image for comparison, and use of level sets mostly takes place in `skimage.segmentation.morphsnakes`.